### ISSUE TITLE
Improve SEO for FlowForge.com

### ISSUE THREAD

**Issue by ZJvandeWeg** [flowforge/website#426]
### Description

Allowing users to find us, instead of vice versa is very important for the scalability of the marketing efforts.

This epic collects the efforts to be made in Q1 2023, as SEO is likely a constant consideration.

**Comment by ZJvandeWeg** [comment]
@Pezmc Any update on this? Hubspot still reports over 1000 issues, and we started with 1500-ish. Think that reduction is not quite what I'd expect after a month of effort?

**Comment by Pezmc** [comment]
Performance is [improved across the board](https://flowforgeworkspace.slack.com/archives/C032Q63FGG1/p1678111362422599) and Google itself reports a pass on the Core Web Vitals Assessment (based on real world usage) and good performance (between 80-100) on both mobile and desktop for several key entry points I checked:

 - [flowforge.com](https://pagespeed.web.dev/analysis/https-flowforge-com/jsjvrcne55?utm_source=search_console&form_factor=mobile&hl=en
 - [flowforge.com/pricing](https://pagespeed.web.dev/analysis/https-flowforge-com-pricing/3jbhep5pqy?hl=en&form_factor=mobile)
 - [flowforge.com/docs](https://flowforge.com/docs/)
 - [flowforge.com/features](https://pagespeed.web.dev/analysis/https-flowforge-com-features/79o09339a1?hl=en&form_factor=mobile)
 - [flowforge.com/blog/2023/03/comparing-node-red-dashboards/](https://flowforge.com/blog/2023/03/comparing-node-red-dashboards/)

When making its assessment HubSpot appears to only consider [TBT](https://developer.chrome.com/en/docs/lighthouse/performance/lighthouse-total-blocking-time/) as a metric, rather than combining several metrics, and has set a threshold of 200ms.

Unfortunately, there's no that much more we can do further to improve TBT other than the existing summary on the [performance improvement issue](https://github.com/flowforge/website/issues/401#issuecomment-1448418011). Most possible improvements are related to third party code performance, which we do not have control of, remaining options include:

 - Stop using Hubspot forms
 - Stop using Hubspot tracking
 - Stop using Google Tag manager
 - Stop using PostHog
 - Change the TTL on third party scripts (or on GitHub pages) - neither of which we control.

In short, the items left aren't reasonably actionable and for now, this is as good as we can get on most pages. If you spot anything that _could_ be actioned, or are happy reducing our use of third party tools, please let me know!

**Comment by joepavitt** [comment]
@ZJvandeWeg any particular reason for this being open? We'll always be wanting to improve SEO, but not sure it's worth keeping an issue open for?
